### ISSUE TITLE
Document behaviour with multiple WSGI workers (Gunicorn etc)

### ISSUE THREAD

**Issue by Bartvds** [praekelt/django-multicore#1]
How does this app work with Gunicorn worker processes?

Say we got 2 cores with 2 Gunicorn workers per core, for a total of 4 WSGI processes. What does this mean for django-multicore?

**Comment by hedleyroos** [comment]
Hi. I will have to check that. The question I have to answer is should each worker have its own pool of extra processes or must only the parent process start a pool of extra processes. This repo is still very new and I haven't got to testing that.

**Comment by hedleyroos** [comment]
I've tested it. Each gunicorn worker will iself also start number-of-core sub-processes. This is expected because gunicorn forks the master process. It is also required because each gunicorn worker has no access to the master's pool of sub-processes. If we use pipes for IPC it may be possible to use only one pool amoung all gunicorn workers, but I've since moved to using memory mapping because the overhead of pipes is too large.

Ok, back to results. Running with gunicorn -w 2 starts 11 Python processes in total for me. Each individual gunicorn worker has its own multicore sub-process pool. I proceeded to hit my instance with 2 concurrent clients doing 200 requests sequentially. I was expecting issues because of the sheer amount of Python processes but the converse happened: per-core utilizations was much better, getting close to 100%, and response times were good. FYI I was testing a patched Django Rest Framework list call with an expensive serializer.

In summary:
Single core has 0.85 second response time. 
Multicore with one client has 0.4 second response time.
Multicore with 2 concurrent clients has 0.8 second response time.

**Comment by hedleyroos** [comment]
Updated the readme
